src/diffusers/models/prior_transformer.py

```diff
@@ -58,24 +59,48 @@ def __init__(
         num_embeddings=77,
         additional_embeddings=4,
         dropout: float = 0.0,
+        act_fn: str = "silu",
+        has_pre_norm: bool = False,
+        has_encoder_hidden_states_proj: bool = True,
+        has_prd_embedding: bool = True,
+        has_post_process: bool = True,
+        time_embed_dim: Optional[int] = None,
+        clip_embedding_dim: Optional[int] = None,
+        out_dim: Optional[int] = None,
```

Comment:
```suggestion
        act_fn: str = "silu",
        has_pre_norm: bool = False,
        has_encoder_hidden_states_proj: bool = True,
        has_prd_embedding: bool = True,
        has_post_process: bool = True,
        time_embed_dim: Optional[int] = None,
        clip_embedding_dim: Optional[int] = None,
        out_dim: Optional[int] = None,
```

Let's try to reduce the number of newly added config here before merging. Ok for now